cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/ProgramOptionConstants.java

```diff
@@ -36,4 +36,6 @@
   public static final String WORKFLOW_BATCH = "workflowBatch";
 
   public static final String SCHEDULE_NAME = "scheduleName";
+
+  public static final String CONCURRENT_PROGRAM_RUN = "concurrent.program.run";
```

Comment:
As per comment on the associated jira https://issues.cask.co/browse/CDAP-513 , we do want to be able to toggle concurrent execution on a per-workflow basis. I discussed with @sreevatsanraman and we can specify that option as workflow properties which later on can be overridden by runtime arguments if required. Thoughts?